Assets/Player/PlayerHUD.gd

```diff
@@ -179,6 +182,12 @@ func _on_PlayerCamera_unselected() -> void:
 	prints("_on_PlayerCamera_unselected")
 	self.ui_context = UIContext.NONE
 
+func _on_TabWidget_button_rotate_left_pressed() -> void:
+	emit_signal("button_rotate_left_pressed")
+
+func _on_TabWidget_button_rotate_right_pressed() -> void:
+	emit_signal("button_rotate_right_pressed")
+
```

Comment:
If _on_TabWidget_button_rotate_left_pressed() is already called by signal, is not great to re-send a signal from it.

Signals with Godot is great powers, so come great responsability.

Try to use less as possible, they have cost on performances.